### PR TITLE
Added C369083 - Finance | Results List | Verify that values in "Name"…

### PR DESCRIPTION
**Pull request by KazachenkaTatsiana**
… columns are hyperlink

https://foliotest.testrail.io/index.php?/cases/view/369083

https://jenkins-aws.indexdata.com/job/folioRancher/job/folioTestingTools/job/runCypressTests/5970/console

image

### CODE REVIEW

File: cypress/e2e/finance/name-columns-is-hyperlink.cy.js [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+        });
+
+        Funds.createViaApi(defaultFund).then((fundResponse) => {
+          defaultFund.id = fundResponse.fund.id;
+        });
+      });
+    });
+
+    cy.createTempUser([
+      permissions.uiFinanceViewFiscalYear.gui,
```
Comment (manowarqqq) [inline cypress/e2e/finance/name-columns-is-hyperlink.cy.js:38]: Permissions. code convention

File: cypress/support/fragments/finance/groups/groups.js [hunk 2]
```diff
@@ -174,4 +177,15 @@ export default {
   checkCreatedInList: (GroupName) => {
     cy.expect(Section({ id: 'group-results-pane' }).find(Link(GroupName)).exists());
   },
+
+  searchByName: (name) => {
+    cy.do([searchField.selectIndex('Name'), searchField.fillIn(name), searchButton.click()]);
+  },
+
+  selectGroupWithDetails: (GroupName) => {
```
Comment (manowarqqq) [inline cypress/support/fragments/finance/groups/groups.js:185]: selectGroupByName

File: cypress/support/fragments/finance/financeHelper.js [hunk 3]
```diff
@@ -91,4 +98,24 @@ export default {
...
+
+  clickLedgerButton() {
+    cy.get(ledgerButton).click();
+  },
+
+  clickGroupButton() {
+    cy.get(groupButton).click();
+  },
+
+  clickFunButton() {
```
Comment (manowarqqq) [inline cypress/support/fragments/finance/financeHelper.js:118]: Fund

File: cypress/support/fragments/finance/financeHelper.js [hunk 4]
```diff
@@ -91,4 +98,24 @@ export default {
   getRandomInvoiceNumber: () => {
     return Math.floor(100000000 + Math.random() * 900000).toString();
   },
+
+  waitLoading() {
+    cy.expect([ledgerFiltersPane.exists(), ledgerResultsPane.exists()]);
+  },
+
+  clickFiscalYearButton() {
```
Comment (manowarqqq) [inline cypress/support/fragments/finance/financeHelper.js:106]: Why not interactors?
Just Button('Fiscal year')

the same for rest

File: cypress/support/fragments/finance/financeHelper.js [hunk 5]
```diff
@@ -91,4 +98,24 @@ export default {
   getRandomInvoiceNumber: () => {
     return Math.floor(100000000 + Math.random() * 900000).toString();
   },
+
+  waitLoading() {
```
Comment (manowarqqq) [inline cypress/support/fragments/finance/financeHelper.js:102]: Looks like it should be in Ledgers fragment

File: cypress/support/fragments/finance/funds/funds.js [hunk 6]
```diff
@@ -983,4 +983,8 @@ export default {
   varifyCanNotCreatePlannedBudget: () => {
     cy.expect(cy.expect(Section({ id: 'plannedBudget' }).find(Button('New')).absent()));
   },
+
+  verifyFunLikNameExists: (FundName) => {
```
Comment (manowarqqq) [inline cypress/support/fragments/finance/funds/funds.js:987]: FundLink

File: cypress/support/fragments/finance/ledgers/ledgers.js [hunk 7]
```diff
@@ -1258,4 +1261,17 @@ export default {
...
     });
   },
+
+  waitLoading() {
+    cy.expect([ledgerResultsPaneSection.exists(), ledgersFiltersSection.exists()]);
+  },
+
+  searchByName: (name) => {
+    cy.do([searchField.selectIndex('Name'), searchField.fillIn(name), searchButton.click()]);
+    cy.wait(4000);
```
Comment (manowarqqq) [inline cypress/support/fragments/finance/ledgers/ledgers.js:1271]: why so long timeout?

File: cypress/support/fragments/finance/ledgers/ledgers.js [hunk 8]
```diff
@@ -1258,4 +1261,17 @@ export default {
...
+  },
+
+  ledgerLinkExists: (name) => {
```
Comment (manowarqqq) [inline cypress/support/fragments/finance/ledgers/ledgers.js:1274]: verify

File: cypress/support/fragments/finance/groups/groupDetails.js [hunk 9]
```diff
@@ -0,0 +1,15 @@
+import { PaneHeader, Section, including } from '../../../../../interactors';
```
Comment (manowarqqq) [inline cypress/support/fragments/finance/groups/groupDetails.js:1]: Please add this file to fragments/finance/index.js

**Review by manowarqqq — APPROVED** [review]

File: cypress/e2e/finance/name-columns-is-hyperlink.cy.js [hunk 10]
```diff
@@ -0,0 +1,100 @@
...
+  it(
+    'C369083 - Finance | Results List | Verify that values in "Name" columns are hyperlink (Thunderjet) (TaaS)',
+    { tags: ['extendedPath', 'thunderjet'] },
+    () => {
+      FinanceHelp.clickFiscalYearButton();
+      FiscalYears.waitLoading();
+      FiscalYears.searchByName(defaultFiscalYear.name);
+      FiscalYears.expextFY(defaultFiscalYear.name);
+
+      const FiscalYearDetails = FiscalYears.selectFisacalYear(defaultFiscalYear.name);
```
Comment (IhorBohdan) [inline cypress/e2e/finance/name-columns-is-hyperlink.cy.js:69]: Why you move method to const?I don't understand

File: cypress/support/fragments/finance/financeHelper.js [hunk 11]
```diff
@@ -13,7 +15,11 @@ const chooseAFilterMessage = 'Choose a filter or enter a search query to show re
 const fiscalResultsList = PaneContent({ id: 'fiscal-year-results-pane-content' });
 const ledgerResultList = PaneContent({ id: 'ledger-results-pane-content' });
 const FundResultList = PaneContent({ id: 'fund-results-pane-content' });
-const FiscalYearResultList = PaneContent({ id: 'fiscal-year-results-pane-content' });
+const filtersPane = Pane({ id: including('filters-pane') });
+const fiscalYearButton = filtersPane.find(Button('Fiscal year'));
```
Comment (IhorBohdan) [inline cypress/support/fragments/finance/financeHelper.js:19]: Please dont move full method to const...move only Button, textfield and another

**Review by IhorBohdan — CHANGES_REQUESTED** [review]
Fix my changes

**Review by IhorBohdan — APPROVED** [review]

**Pull request merged**